### PR TITLE
Updated lock file to fix delisted celery version mismatch.

### PR DESCRIPTION
**Pull request by gsnider2195**
# Closes: #DNE
# What's Changed

This issue is blocking release of v1.6.2

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Review by HanlinMiao — APPROVED** [review]

File: poetry.lock [hunk 1]
```diff
@@ -687,13 +687,13 @@ graph = ["objgraph (>=1.7.2)"]
...
-version = "3.2.20"
+version = "3.2.21"
 description = "A high-level Python Web framework that encourages rapid development and clean, pragmatic design."
 optional = false
 python-versions = ">=3.6"
 files = [
-    {file = "Django-3.2.20-py3-none-any.whl", hash = "sha256:a477ab326ae7d8807dc25c186b951ab8c7648a3a23f9497763c37307a2b5ef87"},
-    {file = "Django-3.2.20.tar.gz", hash = "sha256:dec2a116787b8e14962014bf78e120bba454135108e1af9e9b91ade7b2964c40"},
+    {file = "Django-3.2.21-py3-none-any.whl", hash = "sha256:d31b06c58aa2cd73998ca5966bc3001243d3c4e77ee2d0c479bced124765fd99"},
+    {file = "Django-3.2.21.tar.gz", hash = "sha256:a5de4c484e7b7418e6d3e52a5b8794f0e6b9f9e4ce3c037018cf1c489fa87f3c"},
```
Comment (glennmatthews) [inline poetry.lock:696]: Should this be accompanied by a pyproject.toml update (`poetry add django~3.2.21`)?

File: poetry.lock [hunk 2]
```diff
@@ -1376,13 +1376,13 @@ smmap = ">=3.0.1,<6"
...
-version = "3.1.33"
+version = "3.1.34"
 description = "GitPython is a Python library used to interact with Git repositories"
 optional = false
 python-versions = ">=3.7"
 files = [
-    {file = "GitPython-3.1.33-py3-none-any.whl", hash = "sha256:11f22466f982211ad8f3bdb456c03be8466c71d4da8774f3a9f68344e89559cb"},
-    {file = "GitPython-3.1.33.tar.gz", hash = "sha256:13aaa3dff88a23afec2d00eb3da3f2e040e2282e41de484c5791669b31146084"},
+    {file = "GitPython-3.1.34-py3-none-any.whl", hash = "sha256:5d3802b98a3bae1c2b8ae0e1ff2e4aa16bcdf02c145da34d092324f599f01395"},
+    {file = "GitPython-3.1.34.tar.gz", hash = "sha256:85f7d365d1f6bf677ae51039c1ef67ca59091c7ebd5a3509aa399d4eda02d6dd"},
```
Comment (glennmatthews) [inline poetry.lock:1385]: Should this be accompanied by `poetry add GitPython~3.1.34`?

**Review by glennmatthews — APPROVED** [review]

**Comment by glennmatthews** [comment]
Should we `poetry add celery~5.3.4` so as to protect against any inadvertent installation of the 5.3.2 and 5.3.3 versions with the breaking API changes?

**Comment by gsnider2195** [comment]
> > > Should we `poetry add celery~5.3.4` so as to protect against any inadvertent installation of the 5.3.2 and 5.3.3 versions with the breaking API changes?
> > 
> > 
> > They're no longer on pypi (which is why this lock file update was necessary) so I don't think that's a problem. I think we should update our `pyproject.toml` to these latest versions but we should do that in `develop` (this is targeted to `main`)
> 
> We merge `main` back into `develop` anyway so it's OK to do here.

Are you saying we should update the `pyproject.toml` in this PR? This PR is going directly to `main` with the intent of making the release CI pass.

**Review by bryanculver — APPROVED** [review]

**Pull request merged**